config/core/test-configs.yaml

```diff
@@ -55,6 +58,10 @@ file_systems:
     nfs: 'buster-kselftest/20210520.0/{arch}/full.rootfs.tar.xz'
     root_type: nfs
 
+  debian_buster-libcamera_ramdisk:
+    type: debian-staging
+    ramdisk: 'buster-libcamera/20210602.0/{arch}/rootfs.cpio.gz'
+
```

Comment:
@gctucker libcamera uses nfs, could you include it?

Reply:
No it's still configured to use a ramdisk: https://github.com/kernelci/kernelci-core/pull/720/files#
```yaml
  lc-compliance:
    rootfs: debian_buster-libcamera_ramdisk
    pattern: 'lc-compliance/{category}-{method}-{protocol}-{rootfs}-lc-compliance-template.jinja2'
  ```